### DISCUSSION TITLE
这一关怎么操作啊？看不太懂.......

### DISCUSSION THREAD

**Discussion by ghost** [ByteLegend/ByteLegend#70] [General]
![image](https://user-images.githubusercontent.com/90504600/135110720-79046f08-1fd4-4351-8433-9512c612a469.png)

**Comment by ghost** [comment]
安装好了Git之后该怎么整？

**Reply by blindpirate** [reply]
稍后会上线教程。这一关需要你通过Git提交代码并且通过pull request修复目标仓库中的一个笔误，详细步骤恕不能一一给出，你可通过视频来了解：https://search.bilibili.com/all?keyword=pull%20request&from_source=webtop_search&spm_id_from=333.851

**Reply by ghost** [reply]
#谢谢谢谢
